compiler/qsc_hir/src/mut_visit.rs

```diff
@@ -72,6 +72,7 @@ pub fn walk_item(vis: &mut impl MutVisitor, item: &mut Item) {
         ItemKind::Callable(decl) => vis.visit_callable_decl(decl),
         ItemKind::Namespace(name, _) => vis.visit_idents(name),
         ItemKind::Ty(name, _) => vis.visit_ident(name),
+        ItemKind::Export(_, _) => (),
```

Comment:
I think this should visit the ident:

```suggestion
        ItemKind::Export(name, _) => vis.visit_ident(name),
```

Reply:
We discussed this together and found a solution that uses the assigner from lowering to generate new ids for the cloned `Ident`, which seems to have done the trick!